test-integration/test_integration/fixtures/overrides-project/overrides.txt

```diff
@@ -0,0 +1 @@
+numpy==1.26.4
```

Comment:
We’re only testing whether build passes but not which version was installed? Maybe better if we add a `numpy<=1.26.3` here, basically 2 conflicting req

Reply:
I can add an int test for checking the numpy version in the predict method

Reply:
Resolved in [1a0625c](https://github.com/replicate/cog/pull/2264/commits/1a0625c545c577ff43e664df67966c8ffaf996dc)